### DISCUSSION TITLE
Icon for PPSSPP (PSP Emulator)

### DISCUSSION THREAD

**Discussion by DCG81** [elrumo/macOS_Big_Sur_icons_replacements#1257] [Icon request]
Please can someone help me with an Icon for PPSSPP app (PSP Emulator).

**Comment by OMGzilla-git** [comment]
Done. I submitted 2 variants... one with a black background, the other white. Until approved, you can get them off my profile.
https://macosicons.com/u/OMGzilla

**Reply by DCG81** [reply]
thanks a ton, this is perfect :)
